### DISCUSSION TITLE
[求助]超级社死！！！怎么办！？

### DISCUSSION THREAD

**Discussion by novraincn** [share121/inter-knot#331] [其他]
![1f01db4e025f73c6b8f04cb6a9eaf425412505837](https://github.com/user-attachments/assets/1b8b67d4-37ac-4b06-94e2-4ada8476cfe0)
本人昨晚在和长辈友好交流的时候手机里的涩图不小心被看到了，现在想想都能抠出四室一厅，怎么办？！

**Comment by kunlinm** [comment]
用涩图淹没群聊就说号被盗了doge

**Comment by lappslover** [comment]
这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！这是什么，失忆喷雾？喷一下！

**Comment by EllenJoe14** [comment]
给他们发一份（bushi

**Comment by c1iied** [comment]
不当回事，只要你不尴尬就是他们尴尬

**Comment by wozaofale** [comment]
这个委托我接了！

**Comment by starfox11645** [comment]
换个世界生活吧（笑

**Comment by IX-11** [comment]
不必惊慌，这位先生（女士）。有些事情本就是人生的一环，是生命存在的一大特征。希望您不会因为这次的事情而止步不前，未来会有更多的美好在等待着你

**Comment by white-cloth** [comment]
乐子～

**Comment by ShangHuaWenYuan** [comment]
[点❤我❤看❤可❤爱❤伊❤埃❤斯](https://www.bilibili.com/video/BV18E4m1d7fE/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Comment by kano0x0** [comment]
这个委托我接了！-我来当你父母就可以了

**Comment by yoshizawa-kasumi** [comment]
手机给我，我看看（划掉）我帮你向长辈解释

**Comment by fireflys26710** [comment]
问题不大，谁都会有欲望，他们也是过来人，肯定能理解，就是有点尴尬，没关系的，看看你父母会不会主动找你交流，引导你了解正确的性知识，不找你也没事，你自己注意就好了，没事的

**Comment by 15935784260** [comment]
简单，他不找你聊，你也不找他聊，装作无事发生
